### PR TITLE
Document custom game modes from osu!(lazer) in `Game mode`

### PR DESCRIPTION
**Pull request by Tenexxt**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Game_mode/de.md
Comment (Joehuu) [inline wiki/Game_mode/de.md:5]: This is not the correct way of marking translations as outdated, see https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#outdated-translations.

File: wiki/Game_mode/en.md
Comment (Joehuu) [inline wiki/Game_mode/en.md:10]:
```suggestion
To switch between modes, click the `Mode` button at the bottom of the Song Selection screen for osu!(stable), use the buttons on the top of the screen for osu!(lazer), or use these key combinations:
```

File: wiki/Game_mode/en.md
Comment (Joehuu) [inline wiki/Game_mode/en.md:17]:
```suggestion
![Mode selection interface](/wiki/shared/Interface_mode.png "Mode selection interface in osu!(stable)")
```

File: wiki/Game_mode/en.md
Comment (Joehuu) [inline wiki/Game_mode/en.md:19]:
```suggestion
![Mode selection interface](/wiki/shared/Interface_mode_lazer.png "Mode selection interface in osu!(lazer)")
```

File: wiki/Game_mode/en.md
Comment (Joehuu) [inline wiki/Game_mode/en.md:55]:
```suggestion
In osu!(lazer), you can use custom game modes (also called custom rulesets) that introduce a new way to play the game using the beatmap library that osu! offers. Custom game modes, like the main modes, are often inspired by other rhythm games.
```

File: wiki/Game_mode/en.md
Comment (Joehuu) [inline wiki/Game_mode/en.md:57]: Using "here" for links is discouraged IIRC, but couldn't see a reference in the Article Styling Criteria.

```suggestion
The full list of playable custom game modes can be found in [this GitHub discussion](https://github.com/ppy/osu/discussions/13096).
```

**Review by Joehuu — CHANGES_REQUESTED** [review]

File: wiki/Game_mode/en.md
Comment (Tenexxt) [reply wiki/Game_mode/en.md:57]: Ahh, didn't know. Will change right now

Comment (Tenexxt) [reply wiki/Game_mode/en.md]: done

File: wiki/Game_mode/en.md
Comment (Tenexxt) [reply wiki/Game_mode/en.md]: done

File: wiki/Game_mode/en.md
Comment (Tenexxt) [reply wiki/Game_mode/en.md]: done

File: wiki/Game_mode/en.md
Comment (Tenexxt) [reply wiki/Game_mode/en.md]: .

File: wiki/Game_mode/en.md
Comment (Tenexxt) [reply wiki/Game_mode/en.md]: .

File: wiki/Game_mode/de.md
Comment (Tenexxt) [reply wiki/Game_mode/de.md]: fixed (I think, im not 100% sure so ill leave this open for now)

File: wiki/Game_mode/de.md
Comment (Joehuu) [inline wiki/Game_mode/de.md:5]: This is the correct way.
```suggestion
outdated_translation: true
outdated_since: 5830d6cc5e93d6b10ff5c953761854889b85b0df
```

File: wiki/Game_mode/en.md
Comment (Joehuu) [inline wiki/Game_mode/en.md:53]:
```suggestion
## Custom game modes
```

File: wiki/Game_mode/de.md
Comment (Tenexxt) [reply wiki/Game_mode/de.md:5]: Ahh, thanks.
Will remember!

Comment (Tenexxt) [reply wiki/Game_mode/de.md]: Applied changes

File: wiki/Game_mode/en.md
Comment (Tenexxt) [reply wiki/Game_mode/en.md]: Done

File: wiki/Game_mode/de.md
Comment (Tenexxt) [reply wiki/Game_mode/de.md]: Done

**Comment by Joehuu** [comment]
If you still had access to a browser and want to make commits, the best way is documented in https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/GitHub_web-based_editor.

Not sure if this needs rebasing now because I wouldn't want to merge those commits. Just gonna add the label for now.

File: wiki/Game_mode/en.md
Comment (Joehuu) [inline wiki/Game_mode/en.md:55]: Forgot that "you" should be avoided: https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#perspective

**Comment by Tenexxt** [comment]
> If you still had access to a browser and want to make commits, the best way is documented in https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/GitHub_web-based_editor.
> 
> Not sure if this needs rebasing now because I wouldn't want to merge those commits. Just gonna add the label for now.

I could make a new branch and move all of the changes there into 1 commit, close this PR and make a new one with the new branch.
Yea that's what I'll do, tho I'd prefer to do this **after** this is ready to be merged, so everything would still be easily accessible and the discussions wouldn't be split so I could just close this, link it to the new pr so discussions would be available, and then push this to the wiki unless some mistakes would be missed.

**Comment by Tenexxt** [comment]
moving to #12272

**Comment by cl8n** [comment]
joehuu was saying that before it gets merged into `master`, this branch should be rebased to have its commit history improved (see the tooltip on the tag he applied). if you dont know how to or dont want to do this it's fine, we'll take of it, but dont open new prs please. im going to flip the statuses of each pr since the new one hasn't been interacted with yet

**Comment by Tenexxt** [comment]
> joehuu was saying that before it gets merged into `master`, this branch should be rebased to have its commit history improved (see the tooltip on the tag he applied). if you dont know how to or dont want to do this it's fine, we'll take of it, but dont open duplicate prs please. im going to flip the statuses of each pr since the new one hasn't been interacted with yet

Ahh, sorry. i thought that rebasing meant that i needed to make everything into 1 commit, so i just did what i did.
sorry, still new to github.

File: wiki/Game_mode/en.md
Comment (Joehuu) [inline wiki/Game_mode/en.md:55]:
```suggestion
In osu!(lazer), it's possible to manually install custom game modes (also called custom rulesets) via `.dll`. They introduce a new way to play the game using the beatmap library that osu! offers. Custom game modes, like the main modes, are often inspired by other rhythm games.
```

**Review by Joehuu — COMMENTED** [review]
Looks good after this.

**Review by Joehuu — APPROVED** [review]
LGTM, gonna leave for another maintainer to rebase and merge.

File: wiki/Game_mode/de.md
Comment (The-Last-Cookie) [inline wiki/Game_mode/de.md:2]: Is there a reason why only the translations are marked as stubs?

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]

File: wiki/Game_mode/de.md
Comment (Tenexxt) [reply wiki/Game_mode/de.md:2]: Not really, will change

Comment (Tenexxt) [reply wiki/Game_mode/de.md]: done

**Review by The-Last-Cookie — APPROVED** [review]

**Review by peppy — DISMISSED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**
